File: java/80_Remove_Duplicates_from_Sorted_Array_II.java
// https://leetcode.com/problems/remove-duplicates-from-sorted-array-ii/description/
class Solution {
    public int removeDuplicates(int[] nums) {
        int chk = 1;
        for (int i=1;i<nums.length;i++){
            if (chk==1 || nums[i]!=nums[chk-2]){
                nums[chk++] = nums[i];
            }
        }
        return chk;
    }
}

File: go/80_Remove_Duplicates_from_Sorted_Array_II.go
// https://leetcode.com/problems/remove-duplicates-from-sorted-array-ii/description/
func removeDuplicates(nums []int) int {
    chk := 1;
    for i:=1;i<len(nums);i++{
        if chk == 1 || nums[i] != nums[chk-2] {
            nums[chk] = nums[i];
            chk++;
        }
    }
    return chk;
}